app/assets/javascripts/pieces.js

```diff
@@ -2,5 +2,11 @@ $(document).ready(function(){
   $("td").click(function(){
```

Comment:
@ilyakrasnov This whole section was when I was starting down a JS-based road. I left it thinking it would be useful when we rework the board. Should I stash it for later and remove it for now?